docs/semgrep-cloud-platform/bitbucket-pr-comments.md

```diff
@@ -177,7 +178,11 @@ pipelines:
 </Tabs>
 -->
 
-After defining the `BITBUCKET_TOKEN`, you have successfully set up PR comments in Bitbucket Cloud.
+If you are using a cloud-hosted BitBucket plan to run Semgrep, no extra changes are needed to receive PR comments.
+
+### Receive comments in your VPN or on-premise SCM
+
+<ReceiveCommentsScm />
 
 :::info
```

Comment:
Not a big deal, but we have two callouts in a row here; if there's a place we can move the second, that might be helpful

Reply:
Yeah I removed one. Thanks for pointing out!